mkdocs.yml

```diff
@@ -158,24 +169,12 @@ nav:
     - release-notes/2.0.1.md
     - release-notes/2.0.0.md
     - release-notes/1.x.md
-  - Install and set up:
-    - "Install": "install.md"
-    - "Set up": "setup.md"
-  - Upgrade: upgrade.md
-  - User guide:
-    - "Features": "user_guide.md"
-    - views.md
-    - functions.md
-  - pmm.md
-  - How to:
-    - Auto-create pg_stat_monitor for new databases: auto-enable.md
   - Reference:
-    - pg_stat_monitor view reference:
-      - reference.md
-    - configuration.md
-  - Uninstall: uninstall.md
+    - Views: views.md
+    - functions.md
+    - reference.md
+    - copyright.md
+    - trademark-policy.md
```

Comment:
Looks like these two were added to `Reference` section accidentally, no?

Reply:
No, I aligned the structure with other docs where these ones are under reference